public/js/monetization-api-buttons.js

```diff
@@ -55,6 +56,38 @@ function renderReaderIdForm(selector) {
   document.querySelector(selector).appendChild(form);
 }
 
+/**
+ * renderReaderIdForm
+ * Creates a button to fetch entitlements manually
+ * @param {string} selector
+ */
+function renderPublicationIdForm(selector) {
+  const input = document.createElement('input');
+  input.setAttribute('placeholder', 'Paste publicationId here');
+  input.setAttribute('id', 'publicationId');
+  input.setAttribute('value', 'process.env.PUBLICATION_ID');
```

Comment:
@ed-bird This is where the default setting of the `publicationId` happens, using the staticFileRewrite utility at render time.

Reply:
Got it!